Admin/ORM/MediaAdmin.php

```diff
@@ -36,7 +36,7 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
             ->add('context', null, array(
                 'show_filter' => $this->getPersistentParameter('hide_context') !== true,
             ), 'choice', $options)
-            ->add('category', null, array(
+            ->add('category', 'doctrine_orm_choice', array(
```

Comment:
Actually since the problem is with a filter (isn't it), the relevant code is probably [here](https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/3.x/Guesser/FilterTypeGuesser.php#L26)